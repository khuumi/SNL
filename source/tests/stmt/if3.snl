if not is_done
(is_done is x > 7
y is z * 80)
else (w is 39.2
return true)