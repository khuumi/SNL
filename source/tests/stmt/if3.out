If (Unop Not (Id Global is_done)) (Block [(Expr (Assign (Id Global is_done) (Binop (Id Global x) Gt (Int 7)))),
(Expr (Assign (Id Global y) (Binop (Id Global z) Mult (Int 80))))]) (Block [(Expr (Assign (Id Global w) (Float 39.2))),
(Expr (Return (Bool true)))])